traitsui/ui_editors/data_frame_editor.py

```diff
@@ -306,13 +306,13 @@ class DataFrameEditor(BasicEditorFactory):
     show_titles = Bool(True)
 
     #: Optional list of either column ID or pairs of (column title, column ID).
-    columns = List(Either(Str, Tuple(Str, Str)))
+    columns = List(Union(Str, Tuple(Str, Str)))
```

Comment:
I believe these would have previously defaulted to `None`, but now default to `""`.

Reply:
in practice, the default behavior looks the same to me

```
>>> from traits.api import *
>>> class A(HasTraits):
...     cols = List(Either(Str, Tuple(Str, Str)))
...     cols2 = List(Union(Str, Tuple(Str, Str)))
...
>>> a = A()
>>> a.cols
[]
>>> a.cols2
[]
```

Reply:
Ah I see, seems the `List` removes any discrepancies.  Thanks!